modules/vnet/variables.tf

```diff
@@ -170,57 +175,63 @@ variable "route_tables" {
   type = map(object({
     name = string
     routes = map(object({
-      name                   = string
-      address_prefix         = string
-      next_hop_type          = string
-      next_hop_in_ip_address = optional(string)
+      name                = string
+      address_prefix      = string
+      next_hop_type       = string
```

Comment:
Can we add validation for `next_hop_type`, because we know possible values are: `VirtualNetworkGateway`, `VnetLocal`, `Internet`, `VirtualAppliance` and `None` ?